var/spack/repos/builtin/packages/py-dnaio/package.py

```diff
@@ -13,8 +13,11 @@ class PyDnaio(PythonPackage):
     url      = "https://pypi.io/packages/source/d/dnaio/dnaio-0.3.tar.gz"
     git      = "https://github.com/marcelm/dnaio.git"
 
+    version('0.4.2', sha256='fa55a45bfd5d9272409b714158fb3a7de5dceac1034a0af84502c7f503ee84f8')
     version('0.3', sha256='47e4449affad0981978fe986684fc0d9c39736f05a157f6cf80e54dae0a92638')
 
-    depends_on('python@3.4:', type=('build', 'run'))
+    depends_on('python@3.5:', when='@0.4.2:', type=('build', 'run'))
```

Comment:
I don't think this is actually true is it? I think 3.4+ is fine.

Reply:
Argh, I was looking at the file from the master branch. I will back that out.